### PR TITLE
SDPAP-5915: Remove block user account.

### PR DESCRIPTION
**Pull request by henrytranvan**
Jira - https://digital-vic.atlassian.net/browse/SDPAP-5915

**Changes**

- Enable user expiration date
- Create a user notification email template
- Send notification email

### CODE REVIEW

**Comment by vincent-gao** [comment]
hey @henrytranvan , can you please provide some details about this PR, jira link , what changed what has been included etc... so that we can keep a clear record.

File: modules/tide_user_expire/tide_user_expire.install [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+ * @file
+ * Tide User Expire.
+ */
+
+/**
+ * Implements hook_uninstall().
+ */
+function tide_user_expire_uninstall() {
+  \Drupal::configFactory()->getEditable('tide_tide_user_expire.settings')->delete();
+}
```
Comment (MdNadimHossain) [inline modules/tide_user_expire/tide_user_expire.install:13]: I think there is a extra tide in the naming above, shouldn't it be just `tide_user_expire.settings`

**Review by MdNadimHossain — CHANGES_REQUESTED** [review]
@henrytranvan Please add some tests as well

File: modules/tide_user_expire/tide_user_expire.install (hunk 1 above)
Comment (henrytranvan) [reply modules/tide_user_expire/tide_user_expire.install:13]: Hi @MdNadimHossain 
yes, It's updated.

**Review by MdNadimHossain — APPROVED** [review]
thanks for making the changes

**Pull request merged**